torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -13,24 +13,11 @@ namespace {
 namespace onnx_torch = ::torch::onnx;
 namespace onnx = ::ONNX_NAMESPACE;
```

Comment:
Do we need to add any new tests for updated shape inference?

Reply:
I'm creating an item to keep track of testing, i feel it is better to have new set of tests for shape/type inference only. For this PR i'll add a few under the current test_pytorch_onnx_onnxruntime.py test suite for onnx shape inference.

Reply:
I agree that it would make sense to add new set of shape inference tests.